pkg/mimirpb/mimir.proto

```diff
@@ -28,6 +28,15 @@ message WriteRequest {
 
 message WriteResponse {}
 
+message AddBackfillFileRequest {
+  uint32 tenant_id = 1;
```

Comment:
Remove this. Always get the tenant from the context.

Reply:
The reason I need to add tenant here is that the streaming gRPC handler doesn't receive any context parameter (from generated gRPC code).

Reply:
I think that on the receiver side (ingester) you can get the context with `stream.Context()`. I suggest you to explore it.

Reply:
Done.